docs/src/main/sphinx/language/overview.rst

```diff
@@ -0,0 +1,118 @@
+=====================
+SQL statement support
+=====================
+
+The SQL statement support in Trino can be categorized into a number of topics.
+Numerous statements are part of the core engine and therefore available in all
+use cases. On the other hand the details and architecture of the connected data
+sources prevent some SQL functionality.
+
+For example, if the data source does not support any write operations, Trino can
+not implement support for SQL statements like :doc:`/sql/delete`. Similarly if
+the underlying system does have any security concepts SQL statements like
+:doc:`/sql/create-role` can not provided by Trino and the connector.
```

Comment:
This is technically true - however can mislead people into thinking if their MySQL has users and roles support then Trino can manage them too.

Reply:
Reworded and clarified